src/ConnectedMode.UnitTests/Binding/CompositeBindingConfigProviderTests.cs

```diff
@@ -34,34 +37,27 @@ namespace SonarLint.VisualStudio.ConnectedMode.UnitTests.Binding
     public class CompositeBindingConfigProviderTests
     {
         [TestMethod]
-        public void Ctor_InvalidArgs()
```

Comment:
The class is now MEF-created, the arguments will never be null -> deleted the redundant tests.